crypto/dilithium/pqcrystals_dilithium_ref_common/polyvec.c

```diff
@@ -181,26 +227,38 @@ void polyvecl_pointwise_acc_montgomery(ml_dsa_params *params,
 **************************************************/
 int polyvecl_chknorm(ml_dsa_params *params, const polyvecl *v, int32_t bound)  {
   unsigned int i;
-
-  for(i = 0; i < params->l; ++i)
-    if(poly_chknorm(&v->vec[i], bound))
+  for(i = 0; i < params->l; ++i) {
+    if(poly_chknorm(&v->vec[i], bound)) {
       return 1;
-
+    }
+  }
   return 0;
 }
 
 /**************************************************************/
 /************ Vectors of polynomials of length K **************/
 /**************************************************************/
 
+/*************************************************
+* Name:        polyvecl_uniform_eta
+*
+* Description: FIPS 204: Algorithm 33 ExpandS (for vectors k).
+*              Samples vector v with polynomial coordinates whose
+*              coefficients are in [-eta, eta].
+*
+* Arguments:   - ml_dsa_params: parameter struct
+*              - polyveck v: pointer to input vector
```

Comment:
```suggestion
*              - polyveck v: pointer to output vector
```

Reply:
fixed in [a8777cd](https://github.com/aws/aws-lc/pull/2017/commits/a8777cd91adbc6ddacbc0de027fce11eb8d3b4e7)